api/ai/multiple_round_robin_with_adjusted_winner_algorithm/utils.py

```diff
@@ -0,0 +1,33 @@
+from typing import Callable
+
+from api.models.student import Student
+from api.models.team import TeamShell, Team
+
+
+def is_ordered_envy_freeness_up_to_one_item(
+    team_i: Team, team_j: Team, utility_function: Callable[[Student, TeamShell], float]
+) -> bool:
+    """
+    True if value of team_i is less than value of team_j and if we remove 1 student from team_j, the value of team_i is
```

Comment:
I don't need this comment to be an essay, but it feels like it does not describe the thing this function actually does, notably the shenanigans around requirements